docs/examples/llamacpp/README.md

```diff
@@ -0,0 +1,76 @@
+# Deploy Distributed Inference Service with llama.cpp
+
+In this example, we will use LeaderWorkerSet to deploy a distributed
+inference service using [llama.cpp](https://github.com/ggerganov/llama.cpp).    
+llama.cpp began as a project to support CPU-only inference on a single node, but has
+since expanded to support accelerators and distributed inference.
+
+l.md)
```

Comment:
is this added accidentally?

Reply:
Yes, I'll remove ... I'm thinking I should add support for GPUs also next, so maybe I'll do that at the same time!